README.md

```diff
@@ -131,7 +131,7 @@ Alfa will run in any [ECMAScript 2018](https://www.ecma-international.org/ecma-2
 In order to build Alfa, a recent version (>= 12) of [Node.js](https://nodejs.org/) is required in addition to the [Yarn](https://yarnpkg.com/) package manager. Once Node.js and Yarn are installed, go ahead and install the Alfa development dependencies:
 
 ```console
-$ yarn install
+$ yarn install --immutable
```

Comment:
Immutable installs should not be used when developing Alfa as we want lockfile changes to be committed.

Reply:
My reasoning here was that we mostly want to update lockfile when we want to update lockfile, not when we work on unrelated stuff.

Reply:
CI will fail anyway if the lockfile isn't up to date on push so there's no reason at all to do immutable installs when developing. It might actually be harmful as we'll then have to keep in mind that developers should specifically do mutable installs, i.e. omit `--immutable`, when modifying package manifests. Omitting `--immutable` will always do the right when developing.